src/base/Controller.ts

```diff
@@ -482,4 +482,268 @@ export class Controller {
         throw "never";
     }
   }
+
+  async export({
+    namespace,
+    image,
+    imagePullSecret,
+    helm,
+  }: {
+    namespace: string;
+    image: string;
+    imagePullSecret?: string;
+    helm?: boolean;
+  }) {
+    const targetList = await this.installList(namespace);
+
+    const resources = [
+      ConfigMap.export(
+        {
+          name: this.name,
+          namespace,
+        },
+        {
+          data: {
+            installedResources: JSON.stringify(targetList),
+          },
+        },
+      ),
+    ];
```

Comment:
Em um pacote helm esse config map é desnecessário. Só seria útil se a gente tivesse buscando converter a instalação de um formato para o outro (que ainda assim é questionável). Sugiro remover.

Reply:
concordo, mas mantive por motivo de compatibilidade pra se for tentar fazer algum upgrade via CLI depois, devemos retirar mesmo assim?

Reply:
acredito que ele só seria útil ao fazer um "upgrade" de helm para o modelo antigo.

Reply:
desativei o export do ConfigMap pra o export com helm: true